src/Symfony/Component/Security/Core/Authorization/Voter/Voter.php

```diff
@@ -74,6 +74,7 @@ public function vote(TokenInterface $token, $object, array $attributes)
 
     /**
      * Perform a single access check operation on a given attribute, subject and token.
+     * It is safe to assume that $attribute and $subject pass supports method call.
```

Comment:
Oh :) What about:
It is safe to assume that $attribute and $subject already passed the "supports()" method check.